### PR TITLE
update README and comments

### PR DESCRIPTION
**Pull request by jmartin-sul**
@ndushay @atz 

augment README and method comments so that facet_field configuration is more comprehensible for users of this gem.  this was a first pass at this fix, totally happy to take another stab at it or try to clarify what's there if this is unsatisfactory.  but it does at least remove a minor inaccuracy and add some explanation where there was previously none.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,16 +45,16 @@ In your Blacklight controller configuration (usually `CatalogController`), tell
...
+
+
+Add the hierarchy-specific options to the controller configuration (see the facet_tree method for further explanation and some relevant code, as well as the render_hierarchy method for relevant code)
 
     config.facet_display = {
       :hierarchy => {
-        'tag' => [nil]
+        'queue_status' => [['facet'], ':']
       }
     }
```
Comment (ndushay) [inline README.md]: maybe add "reminder:  per above, the full Solr field name is queue_status_facet, and ':' is the delimiter between the values stored in the field"   or something like that?

File: app/helpers/blacklight/hierarchy_helper.rb [hunk 2]
```diff
@@ -60,10 +60,21 @@ def render_selected_qfacet_value(facet_solr_field, item)
   #   config.facet_display = {
   #     :hierarchy => {
   #       'wf' => [['wps','wsp','swp'], ':'],
-  #       'callnum' => [['top_facet'], '/']
+  #       'callnum_top' => [['facet'], '/'],
+  #       'exploded_tag' => [['ssim'], ':']
   #    }
   #  }
-  # then possible hkey values would be 'wf' and 'callnum'
+  # then possible hkey values would be 'wf' and 'callnum'.
```
Comment (ndushay) [inline app/helpers/blacklight/hierarchy_helper.rb]: possible hkey values would be callnum_top  and   exploded_tag  ???

**Comment by ndushay** [comment]
This looks like an improvement to me!  And glad to get rid of @mbklein's  "don't have time to document this right now" comment.

A couple of line notes that I will leave to your editorial taste.

**Comment by jmartin-sul** [comment]
both great points, hopefully the edits i just pushed address them, let me know if they don't.

**Comment by ndushay** [comment]
:+1:

**Comment by mbklein** [comment]
Sweet. :+1:

**Pull request merged**